docs/meetings/collab-cafe/2025.md

```diff
@@ -1,5 +1,38 @@
 # 2025 Collaboration Cafe Notes Archive
 
+## 2025-04-15
+
+### Check-in :raising_hand:
+
+- Min / @minrk / Simula
+- Raniere Silva / @rgaiacs / GESIS
+- Samuel Gaist / @sgaist / Idiap Research Institute
+- Erik Sundell / @consideRatio / Sundell Open Source Consulting AB
+- Angus Hollands / @agoose77 / 2i2c
+
+### Introduce yourself! :wave:
+
+- `agoose77` — Hi all :wave:, I'm an OS Application Engineer at 2i2c, here on behalf of the Jupyter Book team. I'm joining these Collab Cafés following a suggestion from Kirstie Whitaker that the Jupyter Book team share this space: {%preview https://github.com/jupyter-book/team-compass/issues/20#issuecomment-2749522676 %}
```

Comment:
Does the `{%preview ... %}` syntax rely on a plugin?
![image](https://github.com/user-attachments/assets/4a78b5fc-31b8-41f9-86a5-fa62a1567f92)

Reply:
I don't think that'll work in sphinx - should probably just stick with the link

Reply:
fixed in https://github.com/jupyterhub/team-compass/pull/770/commits/1b28b3d7826c2eb32fc3f8f71a1f76f0149399f2